include/ttmlir/Dialect/TTIR/IR/TTIROps.td

```diff
@@ -274,6 +274,13 @@ def TTIR_SqrtOp : TTIR_ElementwiseUnaryOp<"sqrt"> {
     }];
 }
 
+def TTIR_RsqrtOp : TTIR_ElementwiseUnaryOp<"rsqrt"> {
+    let summary = "Eltwise reciprocal square root.";
+    let description = [{
+      Eltwise square root operation.
```

Comment:
nit: wrong description (missing "reciprocal")

Reply:
Thanks for catching this!